app/routes/ranker.tsx

```diff
@@ -105,8 +108,8 @@ export default function Ranker() {
 
     const handleShareClick = () => {
         const content = orderedAlbums
-            .filter((a) => !a.isHidden)
-            .map((a, i) => `${i + 1}. ${a.name}`)
+            .filter((album: any) => !album.isHidden)
```

Comment:
issue: the type of the parameters in these lambdas (`filter` and `map`) are inferred already from `orderedAlbums`, so asserting them as `any` is getting rid of types unnecessarily.